docs/docs-ref-autogen/excel/excelscript/excelscript.rangefont.yml

```
@@ -4,7 +4,25 @@ uid: 'ExcelScript!ExcelScript.RangeFont:interface'
 package: ExcelScript!
 fullName: ExcelScript.RangeFont
 summary: 'This object represents the font attributes (font name, font size, color, etc.) for an object.'
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script sets the font of A1 to Arial 16.
+   */
+  function main(workbook: ExcelScript.Workbook) {
+    // Get A1 on the currentSheet.
```

I recommend changing currentSheet to current worksheet in this comment. If you agree, could you change it throughout and re-run the generation process? currentSheet is used in other code comments. 
```suggestion
    // Get A1 on the current worksheet.
```